src/com/xilinx/rapidwright/timing/CLKSkewRouteDelay.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * 
+ * Copyright (c) 2021 Ghent University. 
+ * All rights reserved.
+ *
+ * Author: Yun Zhou, Ghent University.
+ *
+ * This file is part of RapidWright. 
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+ */
+
+package com.xilinx.rapidwright.timing;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.xilinx.rapidwright.util.Pair;
+
+/**
+ * Creating a CLKSkewRouteDelay instance parses a supplied text file and stores clock skew, route, and delay data into maps to use.
+ * To obtain the text file, please refer to find_clock_tree_template.tcl under $RAPIDWRIGHT_PATH/tcl/rwroute.
+ * When the file is ready, please use "--clkSkew>" option (see {@link Configuration})
+ * to enable RWRoute to use the file for clock skew-aware timing-driven clock routing.
+ */
+public class CLKSkewRouteDelay {
```

Comment:
Since Clk is not an acronym, let's change it to `ClkSkewRouteDelay`, although, perhaps there is a better name?